test/test_gameController.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import null
+from testing_base import *
+from source.BlackJack import *
+
+def test_gameController():
+    d = Deck()
+    p = Player('Player1', 0)
+    d1 = Dealer()
+    bj = BlackJack(d, p, d1, 0)
+
+    assert str(bj) != null
+    
```

Comment:
I believe that you had trouble with the test at first. But in your commit comment, it seems like you got it to work.